modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
+
+Storage class defaults in {VirtProductName} are configured to provide ReadWriteOnce (RWO) access mode
+by default. Therefore, LiveMigration is not enabled by default unless you
+alter the access mode of your virtual machine.
```

Comment:
The access mode is for the storage class, not the virtual machine (as I understand it)

Reply:
Thanks. Should be covered by above change.